### ISSUE TITLE
Add a chrome extension

### ISSUE THREAD

**Issue by jasonLaster** [bvaughn/redux-devtools-filterable-log-monitor#28]
It'd be nice to be able to easily install a chrome extension with this UI.

**Comment by bvaughn** [comment]
I haven't done much work with browser extensions TBH. Sounds like it would be neat though. Any interest in contributing? :)

**Comment by jasonLaster** [comment]
Yup, the [debugger](https://github.com/devtools-html/debugger.html) is coming along. 

Soon we'll have a standalone electron app that will be able to debug chrome + firefox and will support plugins.

**Comment by bvaughn** [comment]
That's sounds cool! I look forward to seeing it. :smile:
